examples/clusterctl-templates/clusterctl-cluster.yaml

```diff
@@ -35,6 +35,7 @@ metadata:
   name: ${CLUSTER_NAME}
   namespace: ${NAMESPACE}
 spec:
+  nodeDrainTimeout: ${NODE_DRAIN_TIMEOUT:-"0s"}
```

Comment:
Please add this variable to docs/getting-started

Reply:
Done